README.md

```diff
@@ -414,15 +414,46 @@ $ clarinet integrate
 
 Make sure that you have a working installation of Docker running locally.
 
-### Deploy contracts to Testnet
+### Interacting with contracts deployed on Mainnet
 
-You can use Clarinet to deploy your contracts to the public Testnet environment for testing and
-evaluation on a blockchain. Use the following command:
+Composition and interactions between protocols and contracts are one of the key innovations in blockchains. Clarinet was designed to handle this sort of interactions.
+
+Before refering to contracts deployed on Mainnet, they should be explicitily be listed as a `requirement` in the manifest `Clarinet.toml`:
```

Comment:
Maybe also mention that `clarinet contracts requirement <contract-id>` can be used to add this.

Reply:
Indeed, I forgot that we had this command. I revisited it the approach, and went with:

```bash
clarinet requirements add <contract-id>
```

so we have more room for other eventual future requirements related subcommands (remove, remap, ?).